include/openthread/netdata.h

```diff
@@ -57,67 +57,18 @@ typedef uint32_t otNetworkDataIterator; ///< Used to iterate through Network Dat
  */
 typedef struct otBorderRouterConfig
 {
-    /**
-     * The IPv6 prefix.
-     */
-    otIp6Prefix mPrefix;
-
-    /**
-     * A 2-bit signed integer indicating router preference as defined in RFC 4191.
-     */
-    signed int mPreference : 2;
-
-    /**
-     * TRUE, if @p mPrefix is preferred.  FALSE, otherwise.
-     */
-    bool mPreferred : 1;
-
-    /**
-     * TRUE, if @p mPrefix should be used for address autoconfiguration.  FALSE, otherwise.
-     */
-    bool mSlaac : 1;
-
-    /**
-     * TRUE, if this border router is a DHCPv6 Agent that supplies IPv6 address configuration.  FALSE, otherwise.
-     */
-    bool mDhcp : 1;
-
-    /**
-     * TRUE, if this border router is a DHCPv6 Agent that supplies other configuration data.  FALSE, otherwise.
-     */
-    bool mConfigure : 1;
-
-    /**
-     * TRUE, if this border router is a default route for @p mPrefix.  FALSE, otherwise.
-     */
-    bool mDefaultRoute : 1;
-
-    /**
-     * TRUE, if this prefix is considered on-mesh.  FALSE, otherwise.
-     */
-    bool mOnMesh : 1;
-
-    /**
-     * TRUE, if this configuration is considered Stable Network Data.  FALSE, otherwise.
-     */
-    bool mStable : 1;
-
-    /**
-     * TRUE, if this border router is able to supply DNS infomration obtained via ND.  FALSE, otherwise.
-     */
-    bool mNdDns : 1;
-
-    /**
-     * TRUE, if this prefix is a Thread Domain Prefix.  FALSE, otherwise.
-     *
-     * Note: Domain Prefix is introduced since Thread 1.2.
-     */
-    bool mDp : 1;
-
-    /**
-     * The Border Agent Rloc.
-     */
-    uint16_t mRloc16;
+    otIp6Prefix mPrefix;           ///< The IPv6 prefix.
+    signed int  mPreference : 2;   ///< A 2-bit signed int preference (`OT_ROUTE_PREFERENCE_*` values).
+    bool        mPreferred : 1;    ///< Whether or not prefix is preferred.
```

Comment:
nit: how about making even more concise (here and below)
```suggestion
    bool        mPreferred : 1;    ///< Whether prefix is preferred.
```